### DISCUSSION TITLE
adjust the "Show title" or "Show startup message"?

### DISCUSSION THREAD

**Discussion by maddin1967** [teeminus/NoTouchScreenFirmware#40] [Q&A]
Hello,
I have the BTT TFT 35_E3_V3.0 on the BTT SKR 1.4 Turbo. It runs great with clipper. Thank you, I had already written off the TFT and wanted to buy a new one.

Is it possible to adjust the "Show title" or "Show startup message"? In the "main.cpp" I have already identified the lines. But I am not sure about the content. How do I get other text in there? How do I have to format it ?

Greetings
Maddin

Translated with www.DeepL.com/Translator (free version)

**Comment by teeminus** [comment]
Those texts are used for debugging purposes. The title is used to identify the firmware version and the startup message is to check that the emulator is working.

Why do you want to change those texts?

**Reply by maddin1967** [reply]
I would like to add the print name to the title. Also, I would be interested in general how to change this.

**Reply by teeminus** [reply]
Changing those texts won't be easy. The title is binary coded to reduce the firmware size. As this firmware does not use the external flash where the original firmware stores all fonts and config etc, this would have to be statically added to the firmware which would increase the firmware size and thinks would be much more complicated.

The startup message has the same data format that Klipper and Marlin are using to interact with the display. To change this text (which only appears a short amount of time before Klipper or Marlin take over the screen), a new "display" sequence needs to be generated.

However, you can always remove those texts from the firmware if you don't like them.

**Reply by maddin1967** [reply]
The text is ok. And then stays in as it is. Thanks anyway
